src/app/index.tsx

```diff
@@ -1,21 +1,15 @@
 import { Auth } from '@supabase/ui';
 import FullPageSpinner from 'components/fullPage/Spinner';
+import useBrowserTitle from 'hooks/useBrowserTitle';
 import * as React from 'react';
-import { useIntl } from 'react-intl';
-import { useTitle } from 'react-use';
 
 const AuthenticatedApp = React.lazy(
     () => import(/* webpackPrefetch: true */ './Authenticated')
 );
 const UnauthenticatedApp = React.lazy(() => import('./Unauthenticated'));
 
 function App() {
-    const intl = useIntl();
-    useTitle(
-        intl.formatMessage({
-            id: 'routeTitle.loginLoading',
-        })
-    );
+    useBrowserTitle('browserTitle.loginLoading');
```

Comment:
New way to update browser title. Eventually that should be worked into a larger pattern (hooks, componenets, etc.) we use for navigation in general.